abipy/abio/factories.py

```diff
@@ -1413,6 +1413,109 @@ def dfpt_from_gsinput(gs_inp, ph_ngqpt=None, qpoints=None, do_ddk=True, do_dde=T
     return multi
 
 
+def conduc_from_inputs(scf_input, nscf_input, tmesh, ddb_ngqpt, eph_ngqpt_fine, sigma_erange):
+    """
+    Returns a list of inputs in the form of a MultiDataset to perform a set of calculations to determine conductivity.
+    This part require a ground state |AbinitInput| and a non self-consistent |AbinitInput|. You will also need
+    a work to get DDB and DVDB since |ConducWork| needs these files.
+
+    Args:
+        scf_input: |AbinitInput| representing a ground state calculation, the SCF performed to get the WFK.
+
+        nscf_input: |AbinitInput| representing a nscf ground state calculation, the NSCF performed to get the WFK.
+            most parameters for subsequent tasks will be taken from this inputs.
+
+        tmesh : The mesh of temperature where we calculate the conductivity
+
+        ddb_ngqpt: the coarse grid of qpoints used to get the DDB and DVDB files in the previously made phonon_work.
+
+        eph_ngqpt_fine: the fine grid of qpoints that will be interpolated.
+    """
+    # Create a MultiDataset from scf input
+    multi = MultiDataset.from_inputs([scf_input])
+
+    # Add 3 times the nscf input at the end of the MultiDataset
+    extension = MultiDataset.replicate_input(nscf_input, 3)
+    multi.extend(extension)
+
+    # Modify the second nscf input to get a task that interpolate the DVDB
```

Comment:
I see that in your work, there's an option to activate/deactivate the intermediate step that interpolates the DVDB and writes the data to file. In our applications, we never read the DVDB from file since it becomes really huge when you have dense q-meshes. I may decide to remove the Fortran code that reads DFPT potential from file as it makes the q-point parallelism more complicated to maintain/improve.